### PR TITLE
sagemaker rst documentation

### PR DESCRIPTION
**Pull request by xiwang129**
## Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Documentation
- [ ] Test, CI, or build
- [ ] None

## Components (if applicable)
- [ ] BentoService (model packaging, dependency management, handler definition)
- [ ] Model Artifact (model serialization, multi-framework support)
- [ ] Model Server (mico-batching, logging, metrics, tracing, benchmark, OpenAPI)
- [ ] YataiService (model management, deployment automation)
- [ ] Documentation

## Checklist:

- [ ] My code follows the bentoml code style, both `./dev/format.sh` and
  `./dev/lint.sh` script have passed
  ([instructions](https://github.com/bentoml/BentoML/blob/master/DEVELOPMENT.md#style-check-and-auto-formatting-your-code)).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] My change requires a change in [bentoml/gallery](https://github.com/bentoml/gallery) example notebooks
- [ ] I have sent a pull request to [bentoml/gallery](https://github.com/bentoml/gallery) to make that change

### CODE REVIEW

**Comment by xiwang129** [comment]
@flosincapite

File: docs/source/deployment/sagemaker.rst [hunk 1]
```diff
@@ -0,0 +1,429 @@
+Tutorial : Building an AWS SageMaker End-to-end Workflow with BentoML
+=====================================================================
+
+This tutorial provides an end-to-end guide using BentoML with AWS SageMaker -- a machine learning model training platform. It demonstrates the workflow of integrating BentoML with SageMaker, including: setting up a SageMaker notebook instance, model training, creating an S3 bucket, uploading the BentoService bundle into S3, and deploying the BentoML packaged model to SageMaker as an API endpoint using the BentoML CLI tool.
```
Comment (flosincapite) [inline docs/source/deployment/sagemaker.rst:4]: "an end-to-end guide to using"

File: docs/source/deployment/sagemaker.rst [hunk 2]
```diff
@@ -0,0 +1,429 @@
...
+For demonstration, this tutorial uses the IMDB movie review sentiment dataset with BERT and Tensorflow 2.0.(please note: the following model is a modification of the original version: https://github.com/kpe/bert-for-tf2/blob/master/examples/gpu_movie_reviews.ipynb)
```
Comment (flosincapite) [inline docs/source/deployment/sagemaker.rst:5]: Tensorflow 2.0 (please [...]).

(note positioning of periods and spaces)

File: docs/source/deployment/sagemaker.rst [hunk 3]
```diff
@@ -0,0 +1,429 @@
...
+
+  * .. code-block:: bash
+
+        pip install bentoml scikit-learn
+
+
+1 Create a SageMaker notebook instance
+---------------------------------------
+
+For model training in SageMaker, we will start by creating a ** notebook instance ** . After logging into the AWS management console -- type SageMaker to launch the service. From the SageMaker dashboard, select Notebook instances. Go ahead enter a notebook name and select the instance type
```
Comment (flosincapite) [inline docs/source/deployment/sagemaker.rst:24]: I would just write,

"Log in to the AWS management console and navigate to Sagemaker."

You can include a screenshot if you want. "type SageMaker" is not an accurate instruction.

File: docs/source/deployment/sagemaker.rst [hunk 4]
```diff
@@ -0,0 +1,429 @@
...
+2 Model Training -- Movie review sentiment with BERT and TensorFlow2
+---------------------------------------------------------------------
+
+Ignore this section for now
+
+
+3 BentoML SageMaker API Endpoints Deployment
+---------------------------------------------
+
+In this section, we will demonstrate on using BentoML to build production-ready API endpoints and deploy it to AWS SageMaker. The core steps are as follow:
```
Comment (flosincapite) [inline docs/source/deployment/sagemaker.rst:73]: as follows

File: docs/source/deployment/sagemaker.rst [hunk 5]
```diff
@@ -0,0 +1,429 @@
...
+
+1. Create a BentoML service file for model prediction 
+2. Create and save a BentoMl packaged model called a BentoService bundle for model deployment
+3. Upload the BentoService bundle to cloud storage like S3 (optional)
+4. Use Bento CLI and its web UI for local testing
+5. Deploy AWS SageMaker API endpoints through Bento CLI
+6. Use AWS CLI for endpoints testing
+
+Note: for AWS SageMaker deployment, you will need the following prerequisites: 1) Install and configure the AWS CLI 2) Install Docker
```
Comment (flosincapite) [inline docs/source/deployment/sagemaker.rst:82]: The 1) and 2) bullet points should be formatted as bullets.

File: docs/source/deployment/sagemaker.rst [hunk 6]
```diff
@@ -0,0 +1,429 @@
...
+
+.. image:: ../sagemaker/static/Screen Shot 2020-06-26 at 1.45.31 PM
+
+.. code-block:: bash
+
+    !bentoml serve Service:20200702134432_033DAB  
+
+    #sample output
+
+    [2020-06-26 13:43:43,372] WARNING - BentoML local changes detected - Local BentoML repository including all code changes will be packaged together with saved bundle created, under './bundled_pip_dependencies' directory. For using a modified version of BentoML for production deployment, it is recommended to set the 'core/bentoml_deploy_version' config to a http location or your BentoML for on github, e.g.: 'bentoml_deploy_version = git+https://github.com/{username}/bentoml.git@{branch}'
```
Comment (flosincapite) [inline docs/source/deployment/sagemaker.rst:252]: Remove the warnings here--they clutter up the output.

**Review by flosincapite — CHANGES_REQUESTED** [review]

File: sagemaker/sagemaker.rst [hunk 7]
```diff
@@ -0,0 +1,429 @@
+Tutorial : Building an AWS SageMaker End-to-end Workflow with BentoML
```
Comment (flosincapite) [inline sagemaker/sagemaker.rst:1]: Delete this file.

**Review by flosincapite — CHANGES_REQUESTED** [review]
Please remove the Python notebook--I think that's supposed to go in the `gallery` repo, right?

**Pull request merged**